lightly/utils/io.py

```diff
@@ -52,9 +52,10 @@ def check_embeddings(path: str):
         RuntimeError
     """
     header = []
-    with open(path, 'r', newline='') as csv_file:
-        reader = csv.reader(csv_file, delimiter=',')
-        header: List[str] = next(reader)
+    # with open(path, 'r', newline='') as csv_file:
```

Comment:
I had that initially but then moved to the open-close pattern as it saves space. Otherwise, I'll have to reformat the code. But I can change it if you prefer the context manager.